src/hotspot/os/bsd/os_bsd.cpp

```diff
@@ -2710,9 +2695,9 @@ OSReturn os::get_native_priority(const Thread* const thread, int *priority_ptr)
   }
 
   errno = 0;
-#if defined(__OpenBSD__) || defined(__FreeBSD__)
+#if defined(__FreeBSD__)
```

Comment:
FWIW, this can be removed as well.  I can do that though.  Same with the code above.

I also wonder who the final #else applies to if Apple, FreeBSD, NetBSD and OpenBSD are all handled here...

Reply:
> FWIW, this can be removed as well. I can do that though. Same with the code above.
> 
> I also wonder who the final #else applies to if Apple, FreeBSD, NetBSD and OpenBSD are all handled here...

Sounds good. Yea, good point about the extra else. It is probably from when the file was originally copied from Linux and can be pruned.